metrics/web/test/analytics/domain/entities/page_name_test.dart

```diff
@@ -1,34 +1,42 @@
-// Use of this source code is governed by the Apache License, Version 2.0 
+// Use of this source code is governed by the Apache License, Version 2.0
 // that can be found in the LICENSE file.
 
 import 'package:metrics/analytics/domain/entities/page_name.dart';
-import 'package:metrics/common/presentation/navigation/constants/metrics_routes.dart';
+import 'package:metrics/common/presentation/navigation/route_configuration/route_configuration.dart';
 import 'package:test/test.dart';
 
 void main() {
   group("PageName", () {
     test(".loginPage value equals to the login route path", () {
+      final expectedValue = RouteConfiguration.login().path;
```

Comment:
🔖 
What do you think? The `value` prefix here confuses a bit I suppose, as we use the `path` of the route instance. 
```suggestion
      final expected = RouteConfiguration.login().path;
```